drivers/storage/portworx/portworx.go

```diff
@@ -546,6 +546,8 @@ func (p *portworx) SetDefaultsOnStorageCluster(toUpdate *corev1.StorageCluster)
 					autoUpdateComponents(toUpdate)) {
 				toUpdate.Status.DesiredImages.CSISnapshotController = release.Components.CSISnapshotController
 			}
+			toUpdate.Status.DesiredImages.CsiLivenessProbe = release.Components.CsiLivenessProbe
```

Comment:
Why not move this to line 541?
We don't want to populate the desired image for this if CSI is not enabled.